plugins/Denizen/scripts/climbing_chalk.dsc

```diff
@@ -1,79 +1,98 @@
 climbing_chalk:
     type: item
-    debug: false
     material: sugar
     allow in material recipes: false
     display name: <dark_aqua>Climbing Chalk
     lore:
-    - <&7>Strengthens your grip.
+    - <gold>Strengthens your grip.
     - <&7>Allows you to climb on fences, inner walls, buttons, chains, and iron bars.
     - <&7>Lasts 5 minutes.
-    - <&7>Sneak + Right Click in air to apply.
+    - <&7><&keybind[key.sneak]> + <&keybind[key.use]> in air to apply.
     recipes:
         1:
             type: shapeless
             output_quantity: 4
             input: calcite
 
+climbing_chalk_config:
+    type: data
+    colors:
+        chalk_sufficient_warning: red
+        chalk_low_warning: red
+        chalk_out_warning: red
+        chalk_applied: aqua
+
```

Comment:
I meant use the color codes in the denizen config, like `<&[base]>` and `<&[emphasis]>`